diff --git a/build_demo_app/myapp/sources/main.cpp b/build_demo_app/myapp/sources/main.cpp
index 9c47b6a..92757b1 100644
--- a/build_demo_app/myapp/sources/main.cpp
+++ b/build_demo_app/myapp/sources/main.cpp
@@ -2,9 +2,11 @@
 
 #include "mylib1.h"
 #include "mylib2.h"
+#include "my_api.h"
 
 int main()
 {
+    my_api_state_t state = MY_API_STATE_READY;
     int x, y, n;
     FILE *f;
 
@@ -14,6 +16,8 @@ int main()
        return 1;
     }
 
+    printf("next state: %d\n", my_api_next_state(state));
+
     printf("myexe calling mylib1_function:\n");
     x = mylib1_function(x);
 
